specification/search/resource-manager/Microsoft.Search/stable/2015-08-19/search.json

```diff
@@ -584,6 +584,49 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Search/searchServices": {
+      "get": {
+        "tags": [
+          "Services"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "operationId": "Services_ListBySubscription",
+        "x-ms-examples": {
+          "SearchListServicesByResourceGroup": { "$ref": "./examples/SearchListServicesBySubscription.json" }
```

Comment:
is this accurate? byResourceGroup?

Reply:
Good catch, updated to SearchListServicesBySubscription